examples/server/README.md

```diff
@@ -1,8 +1,22 @@
-# llama.cpp/example/server
+# LLaMA.cpp HTTP Server
 
-This example demonstrates a simple HTTP API server and a simple web front end to interact with llama.cpp.
+Fast, lightweight, production ready pure C/C++ HTTP server based on [httplib](https://github.com/yhirose/cpp-httplib), [nlohmann::json](https://github.com/nlohmann/json) and **llama.cpp**.
 
-Command line options:
+Set of LLM REST APIs and a simple web front end to interact with llama.cpp.
+
+**Features:**
+ * SOTA LLM inference performance with GGUF quantized models on GPU and CPU
+ * [OpenAI API](https://github.com/openai/openai-openapi) compatibles chat completions and embeddings routes
+ * Continuous batching
+ * KV cache attention
+ * Embedding
+ * Multimodal
+ * API Key security
+ * Production ready monitoring endpoints
+
+The project is under active development, and we are [looking for feedback and contributors](https://github.com/ggerganov/llama.cpp/issues/4216).
+
+**Command line options:**
```

Comment:
```suggestion
**Features:**
 * LLM inference of F16 and quantum models on GPU and CPU
 * [OpenAI API](https://github.com/openai/openai-openapi) compatible chat completions and embeddings routes
 * Parallel decoding with multi-user support
 * Continuous batching
 * Multimodal (wip)
 * API key security
 * Monitoring endpoints

The project is under active development, and we are [looking for feedback and contributors](https://github.com/ggerganov/llama.cpp/issues/4216).

**Command line options:**
```

Is the API key functionality worth mentioning? I thought it is a basic example and likely needs a lot more for real-world applications (like a database with users and keys)

Reply:
Yes definitely not a security integration end 2 end, but at least, we are validating the token we received. Removed